README.md

```diff
@@ -371,8 +371,9 @@ Carbon focused talks from the community:
 
 -   [The Carbon Language: Road to 0.1](https://chandlerc.blog/slides/2024-ndc-techtown-carbon-road-to-0-dot-1),
     NDC {TechTown}
--   [How designing Carbon with C++ interop taught me about C++ variadics and overloads](https://chandlerc.blog/slides/2024-cppnorth-design-stories),
+-   [How designing Carbon with C++ interop taught me about C++ variadics and overloads](https://www.youtube.com/watch?v=8SGMy9ENGz8),
```

Comment:
```suggestion
-   [How designing Carbon with C++ interop taught me about C++ variadics and overloads](https://www.youtu.be/8SGMy9ENGz8),
```

Note, this is the link you get if you click "share" on the video, and most of the youtube links are using this format.